client/blocks/app-banner/index.jsx

```diff
@@ -218,6 +193,45 @@ export class AppBanner extends Component {
 	}
 }
 
+export function getiOSDeepLink( currentRoute, currentSection ) {
+	const baseURI = 'https://apps.wordpress.com/get';
+	const fragment = buildDeepLinkFragment( currentRoute, currentSection );
+
+	if ( fragment.length === 0 ) {
+		return baseURI;
+	}
+
+	return `${ baseURI }#${ fragment }`;
```

Comment:
fun fact! we can merge these into a single expression

```js
return ( fragment.length > 0 )
	? `${ baseURI }#${ fragment }`
	: baseURI;
```

if you like it that is